x/ccv/consumer/ibc_module.go

```diff
@@ -270,18 +270,25 @@ func (am AppModule) OnAcknowledgementPacket(
 }
 
 // OnTimeoutPacket implements the IBCModule interface
+// the CCV channel state is changed to CLOSED
+// by the IBC module as the channel is ORDERED
 func (am AppModule) OnTimeoutPacket(
 	ctx sdk.Context,
 	packet channeltypes.Packet,
 	_ sdk.AccAddress,
 ) error {
-	var data ccv.SlashPacketData
-	if err := ccv.ModuleCdc.UnmarshalJSON(packet.GetData(), &data); err != nil {
-		return sdkerrors.Wrapf(sdkerrors.ErrUnknownRequest, "cannot unmarshal consumer packet data: %s", err.Error())
-	}
+	var slashData ccv.SlashPacketData
+	var vscMaturedData ccv.VSCMaturedPacketData
 
-	if err := am.keeper.OnTimeoutPacket(ctx, packet, data); err != nil {
```

Comment:
We don't stop the consumer chain any more on timeout? Seems like a pretty huge change in behavior. Is this what we're doing now, or a typo?

Reply:
The function was a no-op anyway, there was no way it could return an error.

```golang
func (k Keeper) OnTimeoutPacket(ctx sdk.Context, packet channeltypes.Packet) error {
	return nil
}
```

It has been like that since at least this commit:
https://github.com/cosmos/interchain-security/commit/af0670240406a89923fb7734562b749d5bf0db30#diff-52b901441beea83792e0e8fd2ddc7c6bf5155c707642e9b5bae10327acf7a62dL205

Reply:
Yes, this is consistent with the spec: https://github.com/cosmos/ibc/blob/main/spec/app/ics-028-cross-chain-validation/methods.md#ccv-ccf-toslash1 and https://github.com/cosmos/ibc/blob/main/spec/app/ics-028-cross-chain-validation/methods.md#ccv-ccf-tomat1